pkg/workloads/tf_api/api.py

```diff
@@ -244,8 +244,16 @@ def parse_response_proto_raw(response_proto):
 
 
 def run_predict(sample):
+    request_handler = local_cache.get("request_handler")
+
+    prepared_sample = sample
+    if request_handler is not None and util.has_function(request_handler, "preinference"):
+        prepared_sample = request_handler.preinference(
+            sample, local_cache["metadata"]["signatureDef"]
```

Comment:
Would it be better to pass in `local_cache["metadata"]`, or is there nothing useful in there except `signatureDef`? Same question for `postinference()` below